app/models/ability.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class Ability
+  include CanCan::Ability
+
+  def initialize(user)
+    can :manage, Group, owner_id: user.id
+	  can :read, Group, participating_users: { user_id: user.id }
+   
+  end
```

Comment:
Check the indentation of your file.

Reply:
Thanks i will be more careful with the indentation and spaces issues. I already made this change.